src/PJ_merc.c

```diff
@@ -57,7 +57,7 @@ static LP e_inverse (XY xy, PJ *P) {          /* Ellipsoidal, inverse */
 
 static LP s_inverse (XY xy, PJ *P) {           /* Spheroidal, inverse */
     LP lp = {0.0,0.0};
-    lp.phi = M_HALFPI - 2. * atan(exp(-xy.y / P->k0));
+    lp.phi = -atan(sinh(-xy.y / P->k0));
```

Comment:
lots of negation there. I guess atan(sinh(xy.y / P->k0)) would do ?

Reply:
Yes, the two negations cancel.

Reply:
You guessed right! Nice catch.